### DISCUSSION TITLE
Add Tokenshop Features to Support Shop Features!"

### DISCUSSION THREAD

**Discussion by InfinityofphoenixTimo** [gameserverapp/Platform#908] [Suggestions]
Add the same features that the token shop has to the support shop. This means things like the ability to only buy a package every X days, level requirements, and on which server they are available or blocked. Offering memberships that can currently be bought twice because you cannot limit them is very exhausting and always requires direct support and refunds for double purchases.

**Comment by mvaessen** [comment]
duplcate of #598
